tracing/README.md

```diff
@@ -0,0 +1,188 @@
+Distributed Tracing
+---
+
+Both Helidon SE and Helidon MP use Opentracing API for tracing
+events.
+
+_Eclipse Microprofile Opentracing specification is not yet implemented_
```

Comment:
_The Eclipse MicroProfile OpenTracing specification is not yet implemented._

Note the case and punctuation.